README.md

```diff
@@ -9,6 +9,66 @@ Blox is being delivered as a managed service via the Amazon ECS Console, API and
 Blox schedulers are built using AWS primitives, and the Blox designs and code are open source. If you are interested in learning more or collaborating on the designs, please read the [design](docs/daemon_design.md).
 If you are currently using Blox v0.3, please read the [FAQ](docs/faq.md).
 
+### Project structure
+For an overview of the components of Blox, run:
+
+```
+./gradlew projects
+```
+
+### Testing
+To run the full unit test suite, run:
+
+```
+./gradlew check
+```
+
+This will run the same tests that we run in the Travis CI build.
+
+### Deploying
+To deploy a personal stack:
+- install the [official AWS CLI](https://aws.amazon.com/cli/)
+- create an IAM user with the following permissions:
+
+    ```json
+    {
+        "Version":"2012-10-17",
+        "Statement":[{
+            "Effect":"Allow",
+            "Action":[
+                "s3:*",
+                "lambda:*",
+                "apigateway:*",
+                "cloudformation:*",
+                "iam:*"
+            ],
+            "Resource":"*"
+        }]
+    }
+
+    ```
+
+  These permissions are pretty broad, so we recommend you use a separate, test account.
+- configure the `default` profile with the AWS credentials for the user you created above
```

Comment:
As we talked offline, I don't think we should rely on profile name at all since that is unrelated to any resources and also can change without affecting resources.